cmd/frontend/graphqlbackend/textsearch.go

```diff
@@ -194,6 +194,7 @@ func textSearch(ctx context.Context, searcherURLs *endpoint.Map, repo gitserver.
 		"ExcludePattern":  []string{p.ExcludePattern},
 		"IncludePatterns": p.IncludePatterns,
 		"FetchTimeout":    []string{fetchTimeout.String()},
+		"Language":        p.Languages,
```

Comment:
Doesn't this need to be `"Languages"` too here?

```suggestion
		"Languages":        p.Languages,
```

Reply:
Good catch 💯